odoo/tests/common.py

```diff
@@ -311,6 +311,14 @@ class BaseCase(TreeCase, MetaCase('DummyCase', (object,), {})):
     longMessage = True      # more verbose error message by default: https://www.odoo.com/r/Vmh
     warm = True             # False during warm-up phase (see :func:`warmup`)
 
+    def setUp(self):
+        def iap_jsonrpc(*args, **kwargs):
+            raise exceptions.AccessError("Unavailable during tests.")
+
+        iap_patch = patch('odoo.addons.iap.tools.iap_tools.iap_jsonrpc', iap_jsonrpc)
```

Comment:
The patch object could be a global, there's no reason to create a new patch on every `setUp`.